azext_iot/constants.py

```diff
@@ -44,7 +44,7 @@
 DEVICETWIN_MONITOR_TIME_SEC = 15
 # (Lib name, minimum version (including), maximum version (excluding))
 EVENT_LIB = ("uamqp", "1.2", "1.3")
-CENTRAL_PNP_INTERFACE_PREFIX = "$iotin:"
+COMPONENT_MARKER = "__t"
```

Comment:
nit - I think we should have a more descriptive name here for COMPONENT_MARKER,  like add PnP and the version other wise its a very generic variable name.